cmd/server/assets/users/index.html

```diff
@@ -19,69 +19,74 @@
-        <thead>
-          <tr>
-            <th scope="col">Name</th>
-            <th scope="col" width="300">Email</th>
-            <th scope="col" width="75">Options</th>
-            <th scope="col" width="40"></th>
-          </tr>
-        </thead>
-        <tbody>
-          {{range $users}}
-          <tr>
-            <td>
-              <a href="/users/{{.ID}}">{{.Name}}</a>
-            </td>
-            <td>
-              {{.Email}}
-            </td>
-            <td class="text-center">
-              {{if .CanAdminRealm $currentRealm.ID}}
-              <span class="badge badge-pill badge-primary">Admin</span>
+    <div class="card mb-3 shadow-sm" style="min-width:550px;">
+      <div class="card-header">Users</div>
+      <div class="card-body">
+        <a class=" btn btn-primary" href="/users/new">Create a new user</a>
+        <a class="btn btn-primary disabled" href="/users/import">Import users</a>
+        {{if $users}}
+        <div class="table-responsive mt-3">
+          <table class="table table-bordered table-striped bg-white">
+            <thead>
+              <tr>
+                <th scope="col">Name</th>
+                <th scope="col" width="300">Email</th>
+                <th scope="col" width="75">Options</th>
+                <th scope="col" width="40"></th>
+              </tr>
+            </thead>
+            <tbody>
+              {{range $users}}
+              <tr>
+                <td>
+                  <a href="/users/{{.ID}}">{{.Name}}</a>
+                </td>
+                <td>
+                  {{.Email}}
+                </td>
+                <td class="text-center">
+                  {{if .CanAdminRealm $currentRealm.ID}}
+                  <span class="badge badge-pill badge-primary">Admin</span>
+                  {{end}}
+                </td>
+                <td class="text-center">
+                  {{- /* cannot delete yourself */ -}}
+                  {{if not (eq .ID $currentUser.ID)}}
+                  <a href="/users/{{.ID}}" class="d-block text-danger" data-method="DELETE"
+                    data-confirm="Are you sure you want to remove '{{.Name}}'?" data-toggle="tooltip"
+                    title="Remove this user">
+                    <span class="oi oi-trash" aria-hidden="true"></span>
+                  </a>
+                  {{end}}
+                </td>
+              </tr>
               {{end}}
-            </td>
-            <td class="text-center">
-              {{- /* cannot delete yourself */ -}}
-              {{if not (eq .ID $currentUser.ID)}}
-              <a href="/users/{{.ID}}" class="d-block text-danger" data-method="DELETE"
-                data-confirm="Are you sure you want to remove '{{.Name}}'?" data-toggle="tooltip"
-                title="Remove this user">
-                <span class="oi oi-trash" aria-hidden="true"></span>
-              </a>
+            </tbody>
+          </table>
+          {{if .pages}}
```

Comment:
We shouldn't do it in this PR, but I wonder if this makes sense to be a function of the renderer instead? Then the HTML just becomes:

```html
{{ paginate $pages }}
```

and the renderer builds this HTML. Then we can make it portable across any pages that need pagination.